.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,32 @@
+name: Lint on pull request
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  full_ci:
+    runs-on: ubuntu-18.04
+    steps:
+      # Get the code into the container
+      - name: Checkout
+        uses: actions/checkout@v2
+      # Get dependencies
+      - name: Python Dependency Installation
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
+      - name: Install dotenv
```

Comment:
the `dependencies` in the current `requirements.txt` file are mainly for local testing and packaging the sdk, we don't have any external dependency for the code itself. So you can remove this step.

You will however need a `try except` in the test file where you import `dotenv` and load it . Can you please add the following conditional import in the test file?

```python
try:
    from dotenv import load_dotenv
    load_dotenv()
except:
    pass
```